packages/design-tokens/src/ThemeManager.ts

```diff
@@ -50,7 +50,9 @@ export class ThemeManager<Theme extends BaseTheme = BaseTheme> {
   }
   public applyCurrentTheme = () => {
     if (typeof window !== "undefined") {
-      this.setRootElement(document.documentElement)
+      this.setRootElement(
+        document.querySelector("#kaizen-root") ?? document.documentElement
```

Comment:
This is a little magical for my liking. What about if `ThemeManager` took an optional second argument: `rootElementId`, rather than having a hardcoded `#kaizen-root`?

Reply:
ah that sounds better! I will do that